its/ruling/src/test/java/org/sonar/python/it/RulingHelper.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * SonarQube Python Plugin
+ * Copyright (C) 2012-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.python.it;
+
+import com.sonar.orchestrator.Orchestrator;
+import com.sonar.orchestrator.locator.FileLocation;
+import com.sonar.orchestrator.locator.MavenLocation;
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+public class RulingHelper {
+
+  private static final String SQ_VERSION_PROPERTY = "sonar.runtimeVersion";
+  private static final String DEFAULT_SQ_VERSION = "LATEST_RELEASE";
+
+  static Orchestrator getOrchestrator() {
+    return Orchestrator.builderEnv()
+      .setSonarVersion(System.getProperty(SQ_VERSION_PROPERTY, DEFAULT_SQ_VERSION))
+      .addPlugin(FileLocation.byWildcardMavenFilename(new File("../../sonar-python-plugin/target"), "sonar-python-plugin-*.jar"))
+      .addPlugin(MavenLocation.of("org.sonarsource.sonar-lits-plugin", "sonar-lits-plugin", "0.10.0.2181"))
+      .build();
+  }
+
+  static List<String> bugRuleKeys() {
```

Comment:
I think it might be annoying to manually maintain this list when we'll add new bug rules or deprecate / remove existing ones. 
What about going through all json descriptors and filter rules having `"type": "BUG"` and `"status": "READY"`?

Reply:
Agreed, it would be much cleaner. Surprinsingly, I didn't manage to get this to work in a clean way, I created SONARPY-984 to fix this in a later sprint if that's okay for you.